Modules/Devices/test/Devices.Domain.Tests/Identities/RejectDeletionProcessTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Backbone.BuildingBlocks.Domain;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Domain.Aggregates.Tier;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Modules.Devices.Domain.Tests.Identities.TestDoubles;
+using Backbone.Tooling;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Domain.Tests.Identities;
+public class RejectDeletionProcessTest
+{
+    [Fact]
+    public void Reject_deletion_process_waiting_for_approval()
+    {
+        // Arrange
+        SystemTime.Set(SystemTime.UtcNow);
+        var identity = CreateIdentityWithDeletionProcessWaitingForApproval();
+        var deviceId = DeviceId.Parse("DVC"); // temporary solution until feature-cancellation-of-a... branch is merged
+
+        // Act
+        identity.RejectDeletionProcess(identity.GetDeletionProcessInStatus(DeletionProcessStatus.WaitingForApproval)!.Id, deviceId);
+
+        // Assert
+        identity.Status.Should().Be(IdentityStatus.Active);
```

Comment:
Unnecessary assertion, because (as stated above) the status has never been anything else.